akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/javadsl/TestProbe.scala

```diff
@@ -221,8 +223,18 @@ abstract class TestProbe[M] {
   @InternalApi protected def expectMessageClass_internal[C](max: FiniteDuration, c: Class[C]): C
 
   /**
-   * Same as `receiveN(n, remaining)` but correctly taking into account
-   * the timeFactor.
+   * Receive one message of type `M` within the default timeout as deadline.
+   */
+  def receiveOne(): M
+
+  /**
+   * Receive one message of type `M`. Wait time is bounded by the `max` duration,
+   * with an [[AssertionError]] raised in case of timeout.
+   */
+  def receiveOne(max: Duration): M
+
+  /**
+   * Same as `receiveN(n, remaining)` but using the default timeout as deadline.
```

Comment:
Same as `receiveMessages(n, remaining)`

(had to use different names in scaladsl and javadsl)